config/initializers/i18n-js.rb

```diff
@@ -0,0 +1,30 @@
+# This is coppied from https://github.com/fnando/i18n-js/blob/master/lib/i18n/js.rb
+# As in spree core en.yml there are translations -
+# en:
+#   no: "No"
+#   yes: "Yes"
+# Which become to true and false and those have no #to_sym method
+# TODO - remove this after spree core locales are fixed
```

Comment:
Great. This way it will be super easy to see what needs to be changed after the Spree upgrade.